RBAC multiple resources - DOCS (#8938)
treeverse/lakeFS: docs/security/rbac.md

    ]
}
```
## Multiple Resources Statements

> Starting from version v1.54.0, LakeFS supports specifying multiple resources in a single RBAC statement.

Reviewer: Also add the minimal fluffy version that is required with emphasis for on-prem enterprise users (in the cloud fluffy is managed for them)

To enable this, the resource field should contain a string representing a JSON-encoded list of resources.

```json
{
    "statement": [
        {
            "action": [
                "fs:Read*"
            ],
            "effect": "allow",
            "resource": "[\"arn:lakefs:fs:::repository/repo1\",\"arn:lakefs:fs:::repository/repo2\"]"
        }
    ]
}
```

> Note: Ensure that the list is properly JSON-encoded as a string—using double quotes and escaping as shown—otherwise, the policy will not be parsed correctly.

Reviewer: Please add example preferably using our python-sdk (most common tool) of creating this policy, rendering into an array.
in other words, I am a lakeFS User with Python-SDK, what do i do when creating a policy with multiple resources?

    ]
}
```
## Multiple Resources Statements

> Starting from version v1.54.0, (for on-prem users using fluffy > v0.11.0) LakeFS supports specifying multiple resources in a single RBAC statement.

Reviewer 2: It's a bit unclear that this configuration applies also to cloud. I think this sentence is too difficult. 
Let's split it up! (Also note it's called lakeFS!)
```suggestion
lakeFS supports specifying multiple resources in a single RBAC statement. This is available on lakeFS Cloud, or on lakeFS Enterprise if starting at version lakeFS v1.54.0 and Fluffy v0.12.0
```
I'm off course unsure about the actual Fluffy version number, I bumped the minor version by 1 because it said ">". It's a good idea to say the first version that works (like you did lakeFS) and not the last version that doesn't work (Fluffy >).

> To enable this, the resource field should contain a string representing a JSON-encoded list of resources.

Reviewer 2: ```suggestion
In addition to a single resource, the resource field can contain a string representing a JSON-encoded list of resources.
```

```json
{
    "statement": [
        {
            "action": [
                "fs:Read*"
            ],
            "effect": "allow",
            "resource": "[\"arn:lakefs:fs:::repository/repo1\",\"arn:lakefs:fs:::repository/repo2\"]"
        }
    ]
}
```

> Note: Ensure that the list is properly JSON-encoded as a string—using double quotes and escaping as shown—otherwise, the policy will not be parsed correctly.

Reviewer 2: A tech worker once told me "you use the word note too much". I think that's a problem here, too.
```suggestion
The list must be properly encoded as a JSON string: quote each resource, and escape those quotes as shown. Otherwise, the policy cannot be parsed.
```

> Multi resource policy creation using python-sdk:

Reviewer 2: Not: either make this a sentence, or remove the colon and make it a heating.

    ]
}
```
## Multiple Resources Statements

lakeFS supports specifying multiple resources in a single RBAC statement. This is available on lakeFS Cloud, or on lakeFS Enterprise if starting at version lakeFS v1.54.0 and Fluffy v0.12.0
In addition to a single resource, the resource field can contain a string representing a JSON-encoded list of resources.

```json
{
    "statement": [
        {
            "action": [
                "fs:Read*"
            ],
            "effect": "allow",
            "resource": "[\"arn:lakefs:fs:::repository/repo1\",\"arn:lakefs:fs:::repository/repo2\"]"
        }
    ]
}
```
The list must be properly encoded as a JSON string: quote each resource, and escape those quotes as shown. Otherwise, the policy cannot be parsed.

### Multi-Resource Policy Creation Using Python SDK

Here is how you can leverage Python SDK to create a multiple resource policy:

```python

> admin1Client.auth_api.create_policy(

Reviewer: 1. `admin1Client` ? 
2. Please show import also of lakefs for completness

    policy=models.Policy(
        id='FSBlockMergingToMain',
        statement=[models.Statement(
            effect="deny",

>             resource="[\"arn:lakefs:fs:::repository/repo1/branch/main\",\"arn:lakefs:fs:::repository/repo2/branch/main\"]",

Reviewer: The point is to show the correct usage with serialization, so keep what you did but don't set the resource hardcoded as serialized string, instead use json lib

```
import json
json.dumps(["arn1", "arn2"]) # resource
```
